soc/arm/renesas_da1469x/Kconfig.soc

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2022 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_FAMILY_DA1469X
```

Comment:
The family/line is Renesas' SmartBond and SoC series is DA1469x, isn't it?

Reply:
@andrzej-kaczmarek If you confirm this family name, please rename drivers + bindings, please also rework file tree for `dts/arm/renesas/smartbond` as I'm gonna add a `dts/arm/renesas/rcar` folder to store renesas rcar device trees !

Reply:
@aaillet I'll change family name to `smartbond` and series to `da1469x` and also update driver names to `..._smartbond` as well